cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletEndpointTest.java

```diff
@@ -260,8 +261,10 @@ private HttpServletRequest prepareRequest(HttpSession session, String userTaskId
     EasyMock.expect(request.getSession()).andReturn(session).anyTimes();
     EasyMock.expect(request.getSession(false)).andReturn(session).anyTimes();
     EasyMock.expect(request.getMethod()).andReturn(method).anyTimes();
-    EasyMock.expect(request.getRequestURI()).andReturn(KafkaCruiseControlServletUtils.REQUEST_URI + resource).anyTimes();
+    EasyMock.expect(request.getRequestURI())
+        .andReturn(KafkaCruiseControlServletTestUtils.getDefaultWebServerApiUrlPrefix() + resource).anyTimes();
     EasyMock.expect(request.getParameterMap()).andReturn(params).anyTimes();
+    EasyMock.expect(request.getPathInfo()).andReturn("/" + resource).anyTimes();
```

Comment:
(Not introduced in this patch) Looks like this class has no `EasyMock.verify(XXX)` calls; hence, it is hard to verify if the expected calls are made as many as required. Should we add verify before appropriate EasyMock.reset() calls? -- e.g. for the first one in `testUserTaskParameters`

```
    EasyMock.verify(answerQueryRequest1, parameters1, _mockUUIDGenerator, _mockHttpSession, _mockHttpServletResponse);
```

Reply:
I tried to add `verify` to `KafkaCruiseControlServletEndpointTest.java` and `UserTaskManagerTest.java` and I realized that the test cases need to be changed to pass the `verify` method and it requires some relatively non-trivial change or even re-write some of the unit tests.

So, I suggest not doing it in this PR since it's irrelevant and creating an issue for this task.

Reply:
Issue created: https://github.com/linkedin/cruise-control/issues/1408